docs/development/devguide/tutorial/02-tools/03-setup.md

```diff
@@ -1,3 +1,119 @@
+Move on by running a similar command to install PHP packages:
+
+```sh
+composer install
+```
+
+You should see some progress information about downloaded packages. You will also
+see that `composer` performs some ILIAS specific tasks. These are included into the
+process via the `composer.json` config file. More specifically, ILIAS will fill
+the `public` folder and create artifacts. Artifacts are php-files that are derived
+from the raw source code and need to be refreshed when source code changes.
+
+### Make `public` Available for your Webserver
+
+The public folder is the folder that needs to be exposed via the webserver.
+We assume that you know how to run and publish a PHP application in general.
+Make sure that your webserver exposes the `public`-folder of your ILIAS
+installation and runs PHP scripts in that folder. If you access your location
+via webbrowser and you see some message that you didn't run the setup until now
+you have been successfull and can move on to the next step.
+
+### Create a Configuration
+
+To install ILIAS you need a configuration file that contains basic configuration
+for your installation. Create a copy of the file `components/ILIAS/setup_/minimal-config.json`.
+Open the file in a text editor and adjust it according to your requirements. Have
+a look into [the documentation of the setup](../../../../../components/ILIAS/Setup/README.md#about-the-config-file)
+for additional configuration variables.
+
+### Run the setup
+
+Almost done: Now you can actually run the ILIAS setup in your ILIAS folder via 
+
+```sh
+php cli/setup.php install your-config.json
+```
+
+where `your-config.json` is the path to the config file you have created in the previous
+step.
+
+The setup process should show some progress information and inform you that the setup
+was indeed executed successfully. Afterwards you should be able to open the
+installation with a web browser and login with the default user and passwort combination
+`user: root` and `password: homer`.
+
+### Further Steps
+
+If you are interested in a more in depth explanation of the ilias installation, have
+a look into [the ILIAS installation guide](../../../../configuration/install.de).
+If you want to understand how the setup works, have a look into the [README of the setup](../../../../../components/Setup/README.md).
+
+## What do I Need to Watch Out For? (Dos & Dont's)
+
+* Please be aware that exposing a web application to the internet is not something
+  to be done lightly. This tutorial covers the basics to create a running installation
+  on a local system for development purpose. Make sure to configure your system
+  appropriatly to make sure that no one illegitematly uses your local installation.
+* Certain changes to the code will require to rebuild artifacts. If you e.g. create
+  new GUI classes or install a plugin, parts of the artifacts will need to be
+  refreshed. If in doubt, run `composer install`. This will rebuild static artifacts
```

Comment:
Is there a reason why you are using `composer install` here and not `composer dump-autoload`?

Reply:
I think `composer install` is more general in the sense that it does all the required activities to get required code in place. People would need to understand what has changed in some code update to understand if `dump-autoload` is enough or `install` is required. Would not want to go there in this tutorial. Is there something where `install` would have a disadvantage over `dump-autoload`?

Reply:
Not really. There are only two really small ones which come to my mind. `composer dump-autoload` is a bit faster and it is not that strict regarding PHP versions. When switching between different ILIAS versions locally, I have to pay more attention using the right PHP version when executing `composer install`. But your argument prevails here of course – thanks for the explanation!

Reply:
Thanks for the feedback =)